projects/1.16/assets/craft-and-hunt/craftandhunt/lang/zh_cn.json

```
@@ -0,0 +1,590 @@
+	"item.minecraft.potion.effect.mushroomized": "蘑菇化药水",
+	"item.minecraft.potion.effect.strong_mushroomized": "蘑菇化药水",
+	"item.minecraft.potion.effect.long_mushroomized": "蘑菇化药水",
+	"item.minecraft.splash_potion.effect.mushroomized": "喷溅型蘑菇化药水",
+	"item.minecraft.splash_potion.effect.strong_mushroomized": "喷溅型蘑菇化药水",
+	"item.minecraft.splash_potion.effect.long_mushroomized": "喷溅型蘑菇化药水",
+	"item.minecraft.lingering_potion.effect.mushroomized": "滞留型蘑菇化药水",
+	"item.minecraft.lingering_potion.effect.strong_mushroomized": "滞留型蘑菇化药水",
+	"item.minecraft.lingering_potion.effect.long_mushroomized": "滞留型蘑菇化药水",
+	"craftandhunt.comment_seven.desc": "箭名称",
+	"item.minecraft.tipped_arrow.effect.vampire": "生命汲取之箭",
+	"item.minecraft.tipped_arrow.effect.poison_strike": "剧毒打击之箭",
+	"item.minecraft.tipped_arrow.effect.magic_strike": "魔法打击之箭",
+	"item.minecraft.tipped_arrow.effect.wither_strike": "凋零打击之箭",
+	"item.minecraft.tipped_arrow.effect.wither": "凋零之箭",
+	"item.minecraft.tipped_arrow.effect.fire_strike": "火焰打击之箭",
+	"item.minecraft.tipped_arrow.effect.poison_resistance": "防毒之箭",
+	"item.minecraft.tipped_arrow.effect.bug_ignore": "昆虫激素之箭",
+	"item.minecraft.tipped_arrow.effect.undead_ignore": "亡灵腐臭之箭",
+	"item.minecraft.tipped_arrow.effect.rangestrength": "力量之箭",
+	"item.minecraft.tipped_arrow.effect.goatstrength": "山羊之箭",
+	"item.minecraft.tipped_arrow.effect.strong_vampire": "生命汲取之箭",
+	"item.minecraft.tipped_arrow.effect.strong_poison_strike": "剧毒打击之箭",
+	"item.minecraft.tipped_arrow.effect.strong_magic_strike": "魔法打击之箭",
+	"item.minecraft.tipped_arrow.effect.strong_wither_strike": "凋零打击之箭",
+	"item.minecraft.tipped_arrow.effect.strong_wither": "凋零之箭",
+	"item.minecraft.tipped_arrow.effect.strong_fire_strike": "火焰打击之箭",
+	"item.minecraft.tipped_arrow.effect.strong_poison_resistance": "防毒之箭",
+	"item.minecraft.tipped_arrow.effect.strong_bug_ignore": "昆虫激素之箭",
+	"item.minecraft.tipped_arrow.effect.strong_undead_ignore": "亡灵腐臭之箭",
+	"item.minecraft.tipped_arrow.effect.strong_rangestrength": "力量之箭",
+	"item.minecraft.tipped_arrow.effect.strong_goatstrength": "山羊之箭",
+	"item.minecraft.tipped_arrow.effect.long_vampire": "生命汲取之箭",
+	"item.minecraft.tipped_arrow.effect.long_poison_strike": "剧毒打击之箭",
+	"item.minecraft.tipped_arrow.effect.long_magic_strike": "魔法打击之箭",
+	"item.minecraft.tipped_arrow.effect.long_wither_strike": "凋零打击之箭",
+	"item.minecraft.tipped_arrow.effect.long_wither": "凋零之箭",
+	"item.minecraft.tipped_arrow.effect.long_fire_strike": "火焰打击之箭",
+	"item.minecraft.tipped_arrow.effect.long_poison_resistance": "防毒之箭",
+	"item.minecraft.tipped_arrow.effect.long_bug_ignore": "昆虫激素之箭",
+	"item.minecraft.tipped_arrow.effect.long_undead_ignore": "亡灵腐臭之箭",
+	"item.minecraft.tipped_arrow.effect.long_rangestrength": "力量之箭",
+	"item.minecraft.tipped_arrow.effect.long_goatstrength": "山羊之箭",
+	"item.craftandhunt.guide.name": "猎人指南",
+	"item.craftandhunt.guide.landing": "本指南旨在帮助您了解所有可以狩猎以及从Craf and Hunt制作的装备！",
```

```suggestion
	"item.craftandhunt.guide.landing": "本指南助你了解狩猎工艺中所有你能猎取的盔甲，还有装备制作工艺！",
```
```suggestion
	"item.craftandhunt.guide.landing": "本指南旨在帮助你了解狩猎工艺中你能猎取什么盔甲，制作什么装备！",
```

This guide is here to help you learn about all the armor you can **hunt** for and gear you can **craft** from **Craft** and **Hunt**!